lib/gds_metrics/middleware.rb

```diff
@@ -8,7 +8,20 @@ def initialize(app)
           use GDS::Metrics::Gzip
           use GDS::Metrics::Auth
 
-          use Prometheus::Client::Rack::Collector, registry: Proxy.new
+          rails_label_builder = proc do |env|
```

Comment:
Can we move this into the `if` statement below as it's only needed for rails apps?

Reply:
it can, shall update :)